packages/Discretization/test/PointCloudProblemGenerator/create_ascii_data_from_exodus.py

```diff
@@ -0,0 +1,71 @@
+# This script reads and exodus file and dumps its node coordinates and
+# connectivities of all elements in all element blocks into 2 seperate files
+# to be used by driver test code
+
+import netCDF4
+import numpy
+import sys
+
+# exodus file
+exodus_file = sys.argv[1]
+
+# prefix of output files
+output_prefix = sys.argv[2]
+
+# load the exodus file
+nc = netCDF4.Dataset( exodus_file )
+
+# extract the node coordinates
+x_coords = nc.variables['coordx']
+y_coords = nc.variables['coordy']
+z_coords = nc.variables['coordz']
+
+# get the number of nodes
+num_node = nc.dimensions['num_nodes'].size
+
+# write the coordinates to a file. first value is a global id (starting at 1)
+# next values are the coordinates
+coord_file = open( output_prefix + '_node_coords.dat', 'w' )
+value = str(num_node) + '\n'
+coord_file.write( value )
+for n in xrange(num_node):
+    value = str(n+1) + ' ' + str(x_coords[n]) + ' ' + str(y_coords[n]) + ' ' + str(z_coords[n]) + '\n'
+    coord_file.write( value )
+coord_file.close()
+
+# get the number of element blocks
+num_block = nc.dimensions['num_el_blk'].size
+
+# get the total number of cell
+total_num_cell = nc.dimensions['num_elem'].size
+
+# write the connectivity to a file
+connect_file = open( output_prefix + '_connectivity.dat', 'w' )
+value = str(total_num_cell) + '\n'
+connect_file.write( value )
+for b in xrange(num_block):
```

Comment:
`xrange` is not valid in python 3. I think we should not write code using python 2 anymore.